BUILD.md

```diff
@@ -0,0 +1,87 @@
+# Development Workflow
+
+## Prerequisites
+
+* You have Go 1.13 or above installed on your local host/development machine.
+* You have Docker installed on your local host/development machine. Docker is required for building NDM container images and to push them into a Kubernetes cluster for testing. 
+* You have `kubectl` installed. For running integration tests, you will require an existing single node cluster. Don't worry if you don't have access to the Kubernetes cluster, raising a PR with the NDM repository will run integration tests for your changes against a Minikube cluster.
+
+## Initial Setup
+
+### Fork in the cloud
+
+1. Visit https://github.com/openebs/node-disk-manager
+2. Click `Fork` button (top right) to establish a cloud-based fork.
+
+### Clone fork to local host
+
+Since Go modules are being used, you can clone the repo in any directory on your local system. 
+Create your clone:
+
+```sh
+
+# Note: Here user= your github profile name
+git clone https://github.com/$user/node-disk-manager.git
+
+# Configure remote upstream
+cd node-disk-manager
+git remote add upstream https://github.com/openebs/node-disk-manager.git
+
+# Never push to upstream master
+git remote set-url --push upstream no_push
+
+# Confirm that your remotes make sense:
+git remote -v
+```
```

Comment:
These comes as part of `git-workflow` right?

Reply:
yes!

Reply:
Updated it. PTAL